### PR TITLE
Added setter for ZMQ_XPUB_VERBOSE

### LINKED ISSUE #243: Support setting ZMQ_XPUB_VERBOSE
Add support for setting [ZMQ_XPUB_VERBOSE](http://api.zeromq.org/4-2:zmq-setsockopt#toc58).

### PR DESCRIPTION
**Pull request by jean-airoldie**
Fixes #243.

### CODE REVIEW

File: tests/test.rs [hunk 1]
```diff
@@ -431,6 +431,22 @@ test!(test_getset_plain_password, {
...
+test!(test_zmq_set_xpub_verbose, {
+    let ctx = Context::new();
+    let xpub = ctx.socket(XPUB).unwrap();
+    let sub = ctx.socket(SUB).unwrap();
+
+    xpub.bind("inproc://set_xpub_verbose").unwrap();
+    xpub.set_xpub_verbose(1).unwrap();
+
+    sub.connect("inproc://set_xpub_verbose").unwrap();
+    sub.set_subscribe("topic".as_bytes()).unwrap();
```
Comment (rotty) [inline tests/test.rs]: I'm not sure the test actually tests if `set_xpub_verbose()` worked, from what I gather from the `zmq_setsockopt` manpage on ZMQ_XPBUB_VERBOSE`, the first subscribe message for a topic is always passed to the XPUB socket; I think you should try to subscribe twice, and then see if you actually get two messages.

File: src/lib.rs [hunk 2]
```diff
@@ -814,6 +814,7 @@ impl Socket {
         (get_reconnect_ivl, set_reconnect_ivl) => ZMQ_RECONNECT_IVL as i32,
         (get_reconnect_ivl_max, set_reconnect_ivl_max) => ZMQ_RECONNECT_IVL_MAX as i32,
         (get_backlog, set_backlog) => ZMQ_BACKLOG as i32,
+        (_, set_xpub_verbose) => ZMQ_XPUB_VERBOSE as i32,
```
Comment (rotty) [inline src/lib.rs]: I think this should map to an `bool`, as that's semantically an on-off switch.

File: tests/test.rs (hunk 1 above)
Comment (jean-airoldie) [reply tests/test.rs]: You are right this is not a proper test.

**Comment by jean-airoldie** [comment]
Should be all good now.

**Comment by rotty** [comment]
Nice, thanks!

**Pull request merged**